.github/workflows/README.md

```diff
@@ -0,0 +1,98 @@
+        T="$( grep 'Offline token' "get-offline-token-$U.log" | cut -d ' ' -f 9 )"
+        sed -i "s|^\($U .*\)|\1 $T ssourl apiurl true|" $INPUT_FILE
+    done
+
+* Process
+
+## Convert space separated file into a json file
+
+  `convert_users_list_to_json_v2.sh users_approved.list users_approved.json`  
+
+## Split json file and base64 encode it into 4 separate files  
+
+  `json_split_and_encode_v2.sh users_approved.json 4`
+  The base64 encoded files produced are in part_1.encoded, part_2.encoded, part_3.encoded, part_4.encoded
+
+  Comment: convert_users_list_to_json_v2.sh and json_split_and_encode_v2.sh exist in tests/load-tests/ci-scripts directory next to merge-json.sh script
+
+## Store the above files contents into STAGING_USERS_1, STAGING_USERS_2, STAGING_USERS_3, STAGING_USERS_4 GITHUB secrets
+
+* To validate the secrets -
+
+# Test the loadtest workflow in my forked repo and it should show that the users.json file is
+
+   created and used in the "Prepare Load Test" step
+   (<https://github.com/naftalysh/e2e-tests/actions/workflows/loadtest.yaml>)
+
+> Runs `./ci-scripts/merge-json.sh $STAGING_USERS_1 $STAGING_USERS_2 $STAGING_USERS_3 $STAGING_USERS_4`
+> shell: /usr/bin/bash -e {0}
+> env:
+> ARTIFACT_DIR: /home/runner/work/e2e-tests/e2e-tests/tests/load-test/.artifacts
+> STAGING_USERS_1:***
+> STAGING_USERS_2:***
+> STAGING_USERS_3:***
+> STAGING_USERS_4:***
+>  
+> Decoded JSON data merged and stored in users.json.
+
+# Produce locally the users.json file that is used by the loadtest workflow
+
+   Use part_1.encoded, part_2.encoded, part_3.encoded, part_4.encoded that were created in the process above -
+
+   export STAGING_USERS_1=$(cat part_1.encoded)
+   export STAGING_USERS_2=$(cat part_2.encoded)
+   export STAGING_USERS_3=$(cat part_3.encoded)
+   export STAGING_USERS_4=$(cat part_4.encoded)
+
+   Run the /tests/load-tests/ci-scripts/merge-json.sh script and it will produce the users.json file
+    as it's used in the loadtest.yaml workflow
+   ./merge-json.sh STAGING_USERS_1 STAGING_USERS_2 STAGING_USERS_3 STAGING_USERS_4                                                                                         ─╯
+
+   The output:
+   Decoded JSON data merged and stored in users.json.
+
+   Open file users.json and you can see it's a users json file that's organized as array of users objects
+   [
+    {
+        "username": "test-rhtap-1",
+        "password": "kqwkdqkdoqkwd",
+        "token": "eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJhZDUyMjdhMy1iY2ZkLTRjZjAtYTdiNi0zOTk4MzVhMDg1NjYifQ.",
+        "ssourl": "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token",
+        "apiurl": "https://api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com",
```

Comment:
Please add some placeholder here instead of real hostname here. I know there is no security issue, but still feels internal. Probably it is just me, but if there is no reason for having it here in literal way, please change it.

Reply:
Done